docs/lesson-plans/week-1.md

```diff
@@ -102,42 +95,541 @@ every week may not end up helping you as much as it would for our project, and y
 issues and concepts we won't be covering, however, feel free to use us as a resource for fixing
 and understanding them as they pop up.
 
+#### End Goal
+
+If you're watching the class online, here's where we'll show an example of the final project.
+
 ### What technologies will I be learning?
 
 During this time, we'll be teaching and providing demonstrations for a Java backend 
 using a Vert.x framework and Typescript frontend using a React framework with a PostgreSQL 
-database. The frontend
-
----
-
-What we’ll be doing over this time is building out a single page application for a 
-social networking site (think Reddit). We’ll be focusing on creating a Java/Vertx backend 
-with a React/Redux frontend and some other dependencies on each to help out with some 
-functionality. We’ll cover a lot of things like git/terminal commands, HTML/CSS/Javascript, 
-the idea of a server, HTTP and REST requests, testing, databases, and cloud deployment. 
-Don’t worry if some of that is unfamiliar to you, we’ll be sure to cover it at a fairly 
-high level in our explanations and examples. 
-
-This program is for anyone who is interested in learning about how a modern web 
-application works, but it helps if you’ve written programs before. If you haven’t, or 
-if you haven’t used Java, check out codecademy’s Java course, and it should teach you 
-pretty much everything you’ll need to know about Java before we start. It will probably 
-take a couple hours, but you can split it up over however long you need. And before we 
-start the course, we’ll be adding you all to a Slack channel that you can use if you get 
-stuck or have questions about anything. For those of you who may know Java and programming 
-concepts pretty well, we also encourage you to try out our program if you’re interested 
-and think there’s something you can learn from us. 
-
-Every week, we’ll be building out the concepts taught. You’ll be given access to what 
-we’re calling (for now at least) ‘catch-up code’, which is our implementation of the 
-concepts, and which should show the progression of our app over time. This catch-up 
-code is also to help out those of you who miss a meeting, are stuck trying to figure 
-out what to do next, or have a busy week where you’re not able to do any work, but 
-want to be prepared for the examples that will be shown in the next meeting. We will 
-also be encouraging you to extend the application by implementing ideas for improvement 
-that you come up with or things that we suggest but won’t end up covering in our 
-examples. Also, if you decide that you want to take your project in a direction 
-different from where we decide to take ours, or even want to work on your own idea, 
-we’ll be here to help you work through issues and provide you with a lot of the concepts 
-you’ll need to build it.
+database. 
+
+The frontend will use packages such as [Material-UI](https://material-ui.com/) for
```

Comment:
It might be helpful to briefly explain front-end and back-end first since we're assuming no web dev knowledge.

Reply:
second that. Just one sentence is fine

Reply:
Threw in an explanation that ended up being way longer, let me know your thoughts.